src/pages/AnimeOnsen/style.less

```diff
@@ -0,0 +1,32 @@
+@import './../pages';
```

Comment:
The styling for mal-sync-active is missing. It is added on the episode list signalising the last watched episode or watched episodes. Normaly it changes the backgroundcolor of the element. But outline would be a possibility here too.
![image](https://user-images.githubusercontent.com/12599334/152619639-3736126b-9549-4de0-a545-4281e4fad32a.png)